hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/ReplicatedMapProxy.java

```diff
@@ -432,4 +441,8 @@ public String toString() {
     public LocalReplicatedMapStats getReplicatedMapStats() {
         return service.createReplicatedMapStats(name);
     }
+
+    private void ensureQuorumPresent() {
```

Comment:
This should probably check if the quorum is of type `READ`.

Reply:
Damn, thx. Fixed